iroha_client_cli/src/main.rs

```diff
@@ -33,20 +34,23 @@ fn main() {
             maintenance::build_app(),
             )
         .get_matches();
-    if let Some(configuration_path) = matches.value_of(CONFIG) {
-        println!("Value for config: {}", configuration_path);
-    }
+    let configuration_path = matches
```

Comment:
What about default value?

Reply:
It has already has default value, so this will fail only if clap fails:
```rust
fn main() {
    let matches = App::new("Iroha CLI Client")
        .version("0.1.0")
        .author("Nikita Puzankov <puzankov@soramitsu.co.jp>")
        .about("Iroha CLI Client provides an ability to interact with Iroha Peers Web API without direct network usage.")
        .arg(
            Arg::with_name(CONFIG)
            .short("c")
            .long(CONFIG)
            .value_name("FILE")
            .help("Sets a config file path.")
            .takes_value(true)
            .default_value("config.json"),
            )
```

Reply:
sorry - didn't see that in diff)